### DISCUSSION TITLE
Interest in Smart meeting notes GSoC

### DISCUSSION THREAD

**Discussion by Mu-Magdy** [openvinotoolkit/openvino#16225] [Google Summer of Code]
Hello OpenVINO Team,

I am excited to apply for the Google Summer of Code program and have been researching potential organizations that align with my skills and interests. I believe that my technical skills in machine learning, deep learning, and data analysis would make me a valuable contributor to your organization's projects.

I have experience in a variety of programming languages, including Python, C++, and SQL. I am proficient in using popular machine learning and deep learning frameworks such as TensorFlow, Keras, and Scikit-learn, and I have completed several projects related to computer vision, and data analytics.

I was involved in building an Arabic Sign Language Translator, where I created a dataset of videos and implemented a deep learning model to achieve high accuracy.

Moreover, I have completed various data science projects that demonstrate my ability to analyze complex data, develop predictive models, and communicate insights effectively. My experience and skills make me confident in my ability to contribute to your organization's projects and make a meaningful impact.

I want to work on (smart meeting notes ) project I think it matches my experience and I will do a good job working on it.

I look forward to the opportunity to work with your organization and contribute to your projects.

**Comment by adrianboguszewski** [comment]
@eaidova, this is related to your project :)

**Reply by eaidova** [reply]
Hello @Mu-Magdy, do you select pre-requisite task https://github.com/openvinotoolkit/openvino_notebooks/issues/832?
Sorry,  can not find your name among assigned models...

**Comment by sawradip** [comment]
I am also  interested in this project. @eaidova . Is there any specific steps that would expose me to  some parts of this  project. Specifically like, how  we  want to integrate those different models.
